### PR TITLE
add api sections

### PR DESCRIPTION
**Pull request by shivamSspirit**
add api integration to app

### CODE REVIEW

File: src/api/category.js [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import axios from 'axios'
+
+const CategoryUrl = '/api/categories';
+
+
+
```
Comment (HeyNitin) [inline src/api/category.js:6]: why is there so much space between codes? it affects the readability, you can reduce this

File: src/api/watchlater.js [hunk 2]
```diff
@@ -0,0 +1,58 @@
+import axios from 'axios'
+
+const watchLaterUrl = '/api/user/watchlater';
+const encodedToken = localStorage.getItem('token')
+
+//  console.log(encodedToken)
```
Comment (HeyNitin) [inline src/api/watchlater.js:6]: you can remove this

**Review by HeyNitin — APPROVED** [review]
code is clean, PR is also very short which is a very good thing, easy to review

**Review by aurobindoGupta — APPROVED** [review]
good work.
have a look at the formatting.

**Pull request merged**